packages/block-library/src/navigation-link/edit.js

```diff
@@ -501,7 +511,7 @@ export default function NavigationLinkEdit( {
 						<>
 							{ ! isInvalid &&
 								! isDraft &&
-								! isLabelFieldFocused && (
+								! isControlFieldFocused && (
```

Comment:
@ellatrix Is it expected that we need to conditionally render RichText like this to avoid it "stealing" focus?